http/src/request/channel/delete_channel_permission_configured.rs

```diff
@@ -23,6 +23,8 @@ impl<'a> DeleteChannelPermissionConfigured<'a> {
         }
     }
 
+    #[deprecated(note = "you've used the request's reason method which is deprecated; \
+                please import the request::AuditLogReason trait")]
```

Comment:
As noted in the discord server it would be nice to have a `since = "0.1.(current version when merged)"` on all these